app/controllers/application_controller.rb

```diff
@@ -240,6 +240,7 @@ def accept_rules_of_use_url
   end
 
   def after_mfa_setup_path
+    user_session.delete(:in_account_creation_flow)
```

Comment:
I'd typically want to avoid a side effect like this in a path getter method if it can be avoided. Is it possible to have this line of code everywhere we call `after_mfa_setup_path`, or is it used in a lot of places?

Reply:
we can do that, it is used in a lot of places, but we can do it